command_line/reindex.py

```diff
@@ -311,19 +312,17 @@ def run(args):
     else:
         change_of_basis_op = sgtbx.change_of_basis_op(params.change_of_basis_op)
 
-    # validate cb_op
-    try:
-        _ = sgtbx.rt_mx(change_of_basis_op.as_xyz())
-    except RuntimeError:
-        sys.exit("Unsuitable change of basis: %s" % params.change_of_basis_op)
-
     if len(experiments):
         space_group = params.space_group
         if space_group is not None:
             space_group = space_group.group()
-        experiments = reindex_experiments(
-            experiments, change_of_basis_op, space_group=space_group
-        )
+        try:
+            experiments = reindex_experiments(
+                experiments, change_of_basis_op, space_group=space_group
+            )
+        except RuntimeError as e:
+            sys.exit("%s" % str(e).split(":")[-1].strip())
```

Comment:
It seems like prefixing "Error" would be slightly more enlightening - or is this a pattern we are avoiding?
```suggestion
            sys.exit("Error: %s" % str(e).split(":")[-1].strip())
```